OK-15098 OK-14539 feat:NFT NPL

linleiqin opened the pull request.

linleiqin committed:
OK-14453 feat:NFT NPL

linleiqin committed:
fix:lint

sunnylqm committed:
Merge branch 'onekey' into feat/NFT_NPL

sidmorizon commented on packages/kit/src/background/services/ServiceNFT.ts:
```diff
@@ -271,6 +272,42 @@ class ServiceNFT extends ServiceBase {
+    items: { contract_address?: string; token_id?: any }[];
+  }) {
+    const apiUrl = `${this.baseUrl}/batchAsset`;
+    const { data, success } = await axios
```
后面是用的 await this.client

linleiqin replied:
fix

linleiqin committed:
fix:axios client

linleiqin committed:
Merge branch 'onekey' into feat/NFT_NPL

linleiqin committed:
Merge branch 'onekey' into feat/NFT_NPL

sunnylqm committed:
Merge branch 'onekey' into feat/NFT_NPL

linleiqin committed:
Merge branch 'onekey' into feat/NFT_NPL

linleiqin committed:
fix:merge NFT Market Modals

linleiqin committed:
fix:color

sunnylqm committed:
Merge branch 'onekey' into feat/NFT_NPL

linleiqin committed:
fix:Merge branch 'onekey' into feat/NFT_NPL

# Conflicts:
#	packages/kit/src/views/NFTMarket/Home/PageHeader/index.tsx

linleiqin committed:
Merge branch 'onekey' into feat/NFT_NPL

franco-chan committed:
fix: merge

franco-chan committed:
style: ui

linleiqin committed:
feat:Calculator

linleiqin committed:
Merge branch 'onekey' into feat/NFT_NPL

franco-chan committed:
style: ui

franco-chan committed:
style: ui

linleiqin committed:
fix:bug

linleiqin committed:
fix:batchAsset bug

linleiqin committed:
Merge branch 'onekey' into feat/NFT_NPL

linleiqin committed:
feat:Add spend value

franco-chan committed:
style: ui

linleiqin committed:
feat:add input spinner

linleiqin committed:
Merge branch 'onekey' into feat/NFT_NPL

linleiqin committed:
feat:add gas fee

rayston92 commented on packages/kit/src/routes/Modal/NFTMarket.tsx:
```diff
@@ -0,0 +1,54 @@
+        headerShown: false,
+        animationEnabled: !!isVerticalLayout,
+      }}
+    >
```
/wtd write these code more efficiently

linleiqin committed:
feat:Save last npl address

linleiqin committed:
Merge branch 'onekey' into feat/NFT_NPL

linleiqin committed:
fix:Merge branch 'onekey' into feat/NFT_NPL

# Conflicts:
#	packages/kit/src/routes/Modal/index.tsx
#	packages/kit/src/routes/Stack/index.tsx
#	packages/kit/src/views/NFTMarket/Home/Stats/Ranking/Container/Mobile.tsx
#	packages/kit/src/views/NFTMarket/NFTAttributesModal/type.ts
#	packages/kit/src/views/NFTMarket/NFTSearchModal/type.ts

linleiqin committed:
fix:lint

linleiqin committed:
fix:lint

linleiqin committed:
fix:npl gas fee

franco-chan committed:
style: i18n and responsive layout

franco-chan committed:
fix: merge

linleiqin committed:
Merge branch 'onekey' into feat/NFT_NPL

linleiqin committed:
fix:yarn.lock

franco-chan committed:
style: calculator ui

franco-chan committed:
fix: merge

linleiqin committed:
Merge branch 'onekey' into feat/NFT_NPL

linleiqin committed:
Merge branch 'onekey' into feat/NFT_NPL

franco-chan committed:
style: share post ui

franco-chan committed:
style: share post

linleiqin committed:
fix:npl data

linleiqin committed:
fix:NPL list data

franco-chan committed:
style: add tooltips

franco-chan committed:
fix: merge

franco-chan committed:
style: fixed tooltips and i18n

franco-chan committed:
style: i18n and ipad layout

linleiqin committed:
fix:format date && Empty NPLCard

franco-chan committed:
style: desktop list style & share post style

linleiqin committed:
fix:iPad UI

linleiqin committed:
fix: merge

# Conflicts:
#	packages/kit/src/routes/Modal/NFTAttributeFilter.tsx
#	packages/kit/src/routes/Modal/SearchNFTCollection.tsx

linleiqin committed:
fix:hide share button on web

revan-zhang commented on packages/components/src/Icon/react/illus/ArrowBottomLeft.tsx:
```diff
@@ -1,6 +1,12 @@
+    viewBox="0 0 38 62"
+    fill="none"
+    stroke="currentColor"
+    accessibilityRole="image"
```
这块的 accessibilityRole 又加回来了 cc @sidmorizon

linleiqin replied:
@franco-chan 看下是不是你加的？

linleiqin committed:
fix:merge

linleiqin committed:
fix:lint
